pkg/apis/messaging/internal/constants.go

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package internal
+
+// StoredChannelVersion is the version of the Channel that is stored in the API server.
+const StoredChannelVersion = "v1alpha1"
```

Comment:
Just to confirm, this is the stored version of the PubSub Channel CRD, right? Not the stored version of the Channel CRD from Knative Eventing?

How will we ensure that when the stored version changes, this constant is also updated? Do we need a comment in the CRD yaml?

Reply:
Yes, this is the version of the PubSub channel stored in the API server. Added a comment to the YAML.